src/api/app/controllers/webui/package_controller.rb

```diff
@@ -114,20 +114,10 @@ def create
 
   def update
     authorize @package, :update?
-    respond_to do |format|
-      if @package.update(package_details_params)
-        format.html do
-          flash[:success] = 'Package was successfully updated.'
-          redirect_to package_show_path(project: @project, package: @package)
-        end
-        format.js { flash.now[:success] = 'Package was successfully updated.' }
-      else
-        format.html do
-          flash[:error] = 'Failed to update package'
-          redirect_to package_show_path(project: @project, package: @package)
-        end
-        format.js
-      end
+    if @package.update(package_details_params)
```

Comment:
I think it would be nice if you would leave this inside the `respond_to` just with the `format.js` block so it's clear what is happening.

Reply:
Done. Take a look again.